packages/replay-next/components/inspector/values/shared.module.css

```diff
@@ -46,7 +46,7 @@
   color: var(--value-type-undefined);
 }
 
-.IconButton {
+.Iconˀtton {
```

Comment:
This is a syntax error.

Reply:
The changes to `KeyValueRenderer` are just to extract the Header part of it outside the component itself, so that it can also be used with the `FunctionRenderer`

Reply:
Ah, will fix this syntax error

Reply:
> The changes to KeyValueRenderer are just to extract the Header part of it outside the component itself, so that it can also be used with the FunctionRenderer

I see.

I think the thing that's throwing me about that component is the name.

It isn't really a "key value **header**" – it's not shown "above" the key value component (like a header). It's shown in between an expanded value and its properties list. It seems more like on of the properties?

Reply:
Really it's just another key-value pair, where the value is a link.

I don't think we really need to share that code.